setup.py

```diff
@@ -1,17 +1,16 @@
-from setuptools import setup
+from setuptools import setup, find_packages
 
 setup(name='lsm',
       version='0.1',
       description='Liquid state machines (LSM) for spiking neural networks (SNN).',
-      #  url='...',  # TODO
+      url='https://github.com/HBPNeurorobotics/LSM',
```

Comment:
In this repository, it would make more sense for the url to be  'https://github.com/IGITUGraz/LSM'

Reply:
that's absolutely right - to be honest I thought the pull request was against the HBP Neurorobotics repo, but all good ;-)